libs/libfolly/patches/103-arm-yield.patch

```diff
@@ -0,0 +1,11 @@
+--- a/folly/portability/Asm.h
++++ b/folly/portability/Asm.h
+@@ -38,7 +38,7 @@ inline void asm_volatile_pause() {
+   ::_mm_pause();
+ #elif defined(__i386__) || FOLLY_X64
+   asm volatile("pause");
+-#elif FOLLY_AARCH64 || defined(__arm__)
++#elif FOLLY_AARCH64 || (defined(__arm__) && !(__ARM_ARCH < 7))
```

Comment:
seems like boost also ran into this issue and https://github.com/boostorg/fiber/commit/6f152b6e00d107e0abde22954d1da271cb98a18d fixed it for them.
This looks good I think.

Reply:
Hmm that commit solves it differently. Maybe it makes sense to make a FOLLY_ARM define and mirror the platforms in the boost commit. Maybe even add

asm volatile ("nop" ::: "memory"); for nonsupported platforms.